administrator/language/de-DE/com_newsfeeds.sys.ini

```diff
@@ -6,9 +6,9 @@
 
 COM_NEWSFEEDS="Newsfeeds"
 COM_NEWSFEEDS_CATEGORIES="Kategorien"
-COM_NEWSFEEDS_CATEGORIES_VIEW_DEFAULT_DESC="Zeigt alle Kategorien in einer Newsfeed-Kategorien an."
+COM_NEWSFEEDS_CATEGORIES_VIEW_DEFAULT_DESC="Zeigt alle Kategorien in einer Newsfeed-Kategorie an."
```

Comment:
Hier eventuell auch `Zeigt eine Liste aller Kategorien in einer ...` (wie in com_content)?